src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetGeneralTitle/index.spec.tsx

```diff
@@ -0,0 +1,292 @@
+      },
+      method: 'PATCH',
+      body: JSON.stringify({
+        allow_recording: true,
+      }),
+    });
+    expect(report).not.toHaveBeenCalled();
+  });
+
+  it('clicks on the toggle button for deactivate the live recording', async () => {
+    const mockedVideo = videoMockFactory({
+      id: 'video_id',
+      allow_recording: true,
+    });
+
+    fetchMock.patch('/api/videos/video_id/', {
+      ...mockedVideo,
+      allow_recording: false,
+    });
+
+    const queryClient = new QueryClient();
+
+    render(
+      wrapInIntlProvider(
+        <QueryClientProvider client={queryClient}>
+          <DashboardVideoLiveWidgetGeneralTitle video={mockedVideo} />,
+        </QueryClientProvider>,
+      ),
+    );
+
+    screen.getByText('Activate live recording');
+    const recordingToggleButton = screen.getByRole('checkbox', {
+      name: 'Activate live recording',
+    });
+
+    await act(async () => userEvent.click(recordingToggleButton));
+    expect(fetchMock.calls()).toHaveLength(1);
+    expect(fetchMock.lastCall()![0]).toEqual('/api/videos/video_id/');
+    expect(fetchMock.lastCall()![1]).toEqual({
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      method: 'PATCH',
+      body: JSON.stringify({
+        allow_recording: false,
+      }),
+    });
+    expect(report).not.toHaveBeenCalled();
+  });
+
+  it('clicks on the toggle button for activate live recording, but backend returns an error', async () => {
+    const mockedVideo = videoMockFactory({
+      id: 'video_id',
+      allow_recording: false,
+    });
+
+    fetchMock.patch('/api/videos/video_id/', 500);
+
+    const queryClient = new QueryClient();
+
+    render(
+      wrapInIntlProvider(
+        <QueryClientProvider client={queryClient}>
+          <Toaster />
+          <DashboardVideoLiveWidgetGeneralTitle video={mockedVideo} />,
+        </QueryClientProvider>,
+      ),
+    );
+
+    screen.getByText('Activate live recording');
+
+    const recordingToggleButton = screen.getByRole('checkbox', {
+      name: 'Activate live recording',
+    });
+    await act(async () => userEvent.click(recordingToggleButton));
+
+    expect(fetchMock.calls()).toHaveLength(1);
+    expect(fetchMock.lastCall()![0]).toEqual('/api/videos/video_id/');
+    expect(fetchMock.lastCall()![1]).toEqual({
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      method: 'PATCH',
+      body: JSON.stringify({
+        allow_recording: true,
+      }),
+    });
+    screen.getAllByText('Recording activation status update has failed !');
+  });
+});
+
+describe('test text input in <DashboardVideoLiveWidgetGeneralTitle />', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    matchMedia = new MatchMediaMock();
+  });
+  afterEach(() => {
+    fetchMock.restore();
+    matchMedia.clear();
+  });
+
+  it('types some text in an empty input text', async () => {
+    const mockedVideo = videoMockFactory({
+      id: 'video_id',
```

Comment:
```suggestion
```

And then use `video.id` everywhere you need it.

Reply:
done